### ISSUE TITLE
WCAG 1.3.1: Ensure <li> elements are used semantically (.Header-profileWrapper)

### ISSUE THREAD

**Issue by Erikapp** [Erikapp/accessibility-test#31]
#### Issue
`Ensure <li> elements are used semantically` ([listitem](https://accessibilityinsights.io/info-examples/web/listitem))
#### Target application
[Aeroméxico](https://www.aeromexico.com/pt-br)
#### Element path
.Header-profileWrapper
#### Snippet
`<li class="Header-profileWrapper">`
#### How to fix
    Fix any of the following:
      List item does not have a <ul>, <ol> parent element
#### Environment
Chrome version 137.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
